master/buildbot/worker/latent.py

```diff
@@ -53,6 +56,16 @@ def reconfigService(self, name, password,
         self.build_wait_timeout = build_wait_timeout
         return AbstractWorker.reconfigService(self, name, password, **kwargs)
 
+    def getRandomPass(self):
+        """
+        compute a random password
+        There is no point to configure a password for a LatentWorker, as it is created by the master.
+        For supporting backend, a password can be generated by this API
+        """
+        return ''.join(
+            random.choice(string.ascii_uppercase + string.digits)
```

Comment:
What I've done for this is to use `str(uuid.uuid4().hex)`.

This method will create all uppercase+digit passwords. Seems like `string.ascii_letters` would be a better choice in order to get lowercase, uppercase, and digits.